app/components/DocumentMeta.js

```diff
@@ -61,8 +62,20 @@ function DocumentMeta({
     deletedAt,
     isDraft,
     lastViewedAt,
+    isTasks,
+    tasksPercentage,
+    tasksMessage,
   } = document;
 
+  const progressBar = React.useMemo(() => {
+    return (
+      <>
+        <CircularProgressBar percentage={tasksPercentage} />
+        &nbsp;{tasksMessage}
```

Comment:
Let's move `tasksMessage` inline here so that it can be translated, I prefer the english language version used on GitHub as an inspiration as 1/3 might not be immediately clear in the context of Outline.

### Suggested variants

- `complete=0`: "X tasks"
- `complete>0 && complete<total`: "X of X tasks"
- `complete===total`: "X tasks done"